src/main/java/com/jalasoft/selenium/bruno/movies/Customer.java

```diff
@@ -0,0 +1,116 @@
+package com.jalasoft.selenium.bruno.movies;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+/**
+ * This class represents a Customer.
+ */
+class Customer {
+    private static final String BRAKE_LINE = "\n";
+    private static final String TABULATION = "\t";
+    private String customerName;
+    private List<Rental> rentalList = new ArrayList<>();
+
+    /**
+     * Constructor of a Custumer.
+     *
+     * @param name of customer.
+     */
+    Customer(final String name) {
+        customerName = name;
+    }
+
+    /**
+     * This method adds a new rental.
+     *
+     * @param newRental to be added.
+     */
+    public void addRental(final Rental newRental) {
+        rentalList.add(newRental);
+    }
+
+    /**
+     * This method gets the customer name.
+     *
+     * @return customer name.
+     */
+    public String getName() {
+        return customerName;
+    }
+
+    /**
+     * This method creates the ticket.
+     *
+     * @return the ticket
+     */
+    public String statement() {
+
+        StringBuilder result = new StringBuilder();
+        result.append("Rental Record for " + getName() + BRAKE_LINE);
+        for (Rental rentals : rentalList) {
+            result.append(TABULATION + rentals.getMovie().getTitle() + TABULATION);
+        }
+        result.append("Amount owed is " + String.valueOf(calculateTotalAmount())).append(BRAKE_LINE);
+        result.append(BRAKE_LINE);
+        result.append("You earned " + String.valueOf(calculateTotalFrequentRentalPoints())).append(BRAKE_LINE);
+        result.append(" frequent renter points");
+        return result.toString();
+    }
+
+    /**
+     * This method get the frequent renter points.
+     *
+     * @param frequentRenterPoints of a customer.
+     * @param rentals              made by customer.
+     * @return the frequent renter points.
+     */
+    public int getFrequentRenterPoints(final int frequentRenterPoints, final Rental rentals) {
+        int count = frequentRenterPoints;
+        count++;
+        if ((rentals.getMovie().getPriceCode() == 1)
```

Comment:
Why do we still need the getPriceCode method?
you need to take advantage from inheritance implementation here